src/sentry/api/endpoints/organization_pull_request_file_issues.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import annotations
+
+from rest_framework import serializers, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.models.group import Group
+from sentry.models.organization import Organization
+from sentry.snuba.referrer import Referrer
+from sentry.tasks.integrations.github.language_parsers import PATCH_PARSERS
+from sentry.tasks.integrations.github.open_pr_comment import (
+    get_projects_and_filenames_from_source_file,
+    get_top_issues_by_count_for_file,
+)
+
+
+class PullRequestFileSerializer(serializers.Serializer):
```

Comment:
How is this gets related to a PR? Is it by `patch`? Or I can pass any file and get issues for it regardless of PR?